katib/katib-controller/overlays/ibm-storage-config/kustomization.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+bases:
+- ../../base
+patchesStrategicMerge:
+- katib-db-deployment.yaml
+images:
+  - name: mysql
+    newTag: "5.6"
```

Comment:
Is 5.6 feature complete for Katib? The default seems to be at 8.

Reply:
We discussed with the Katib team at https://github.com/kubeflow/katib/issues/725 and they said the katib-db didn't use any mysql 8 specific feature.